examples/vrp-litmus/Dockerfile-vrp

```diff
@@ -0,0 +1 @@
+FROM envoyproxy/envoy-google-vrp-dev:latest
```

Comment:
WDYT about just moving VRP into examples? It is technically an example of a somewhat secure configuration? Would that be simpler and more self contained?

Reply:
Not sure how this would integrate with the DockerHub push flow. Examples run after the base images that get pushed to DockerHub are built in CI. Do we push any other examples to DockerHub?

Reply:
Yeah dunno, just a thought. NBD.